File: Api.Server/src/Services/Center/Module/Dto/QueryModulePagedInput.cs
using Fast.Center.Enum;

namespace Fast.Center.Service.Module.Dto;

/// <summary>
/// <see cref="QueryModulePagedInput"/> 获取模块分页列表
/// </summary>
public class QueryModulePagedInput : PagedInput
{
    /// <summary>
    /// 应用Id
    /// </summary>
    public long? AppId { get; set; }

    /// <summary>
    /// 查看类型
    /// </summary>
    public ModuleViewTypeEnum? ViewType { get; set; }

    /// <summary>
    /// 状态
    /// </summary>
    public CommonStatusEnum? Status { get; set; }
}

File: App.Client/src/api/services/module/models/QueryModulePagedInput.ts
import { ModuleViewTypeEnum } from "@/api/enums/ModuleViewTypeEnum";
import { CommonStatusEnum } from "@/api/enums/CommonStatusEnum";

/**
 * Fast.Center.Service.Module.Dto.QueryModulePagedInput 获取模块分页列表
 */
export interface QueryModulePagedInput extends PagedInput  {
  /**
   * 应用Id
   */
  appId?: number;
  /**
   * 
   */
  viewType?: ModuleViewTypeEnum;
  /**
   * 
   */
  status?: CommonStatusEnum;
}

